crates/compiler/builtins/roc/Num.roc

```diff
@@ -975,10 +976,25 @@ remChecked = \a, b ->
 
 isMultipleOf : Int a, Int a -> Bool
 
+## Does a "bitwise and". Each bit of the output is 1 if the corresponding bit
+## of x AND of y is 1, otherwise it's 0.
 bitwiseAnd : Int a, Int a -> Int a
+
+## Does a "bitwise or". Each bit of the output is 0 if the corresponding bit
```

Comment:
You have your comments out of order. This is Xor the next is Or. Also, the next line still say `AND`

Reply:
Oh my bad, fixed both and pushed.
I will make sure to proofread better next time!